idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/foundation/AnnotatedString.kt

```diff
@@ -0,0 +1,73 @@
+package io.github.composegears.valkyrie.ui.foundation
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+
+data class HighlightColors(
+    val keywordColor: Color,
+    val identifierColor: Color,
+    val defaultColor: Color,
+) {
+    companion object {
+        val DARK = HighlightColors(
+            keywordColor = Color(0xFFC87631),
+            identifierColor = Color(0xFF8B6C9B),
+            defaultColor = Color(0xFFBCBEC4),
+        )
+        val LIGHT = HighlightColors(
+            keywordColor = Color(0xFF4A6EC9),
+            identifierColor = Color(0xFF881395),
+            defaultColor = Color(0xFF1F1F1F),
+        )
+    }
+}
+
+fun getHighlightedCode(
```

Comment:
@Goooler I want to try out https://github.com/SnipMeDev/KodeView for syntax highlight
what do you think? 🙂

Reply:
Or we can try out https://github.com/MohamedRejeb/Compose-Rich-Editor.

I'm using this in my note app.

Reply:
Thanks!

Will try KodeView as it doesn't have any 3rd party dependencies
if it not work as I expect - Compose-Rich-Editor